cni/network/invoker_azure.go

```diff
@@ -86,6 +95,29 @@ func (invoker *AzureIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, er
 	return addResult, err
 }
 
+func (invoker *AzureIPAMInvoker) deleteIpamState() {
+	cniStateExists, err := platform.CheckIfFileExists(platform.CNIStateFilePath)
+	if err != nil {
+		log.Printf("[cni] Error checking CNI state exist: %v", err)
+		return
+	}
+
+	ipamStateExists, err := platform.CheckIfFileExists(platform.CNIIpamStatePath)
+	if err != nil {
+		log.Printf("[cni] Error checking IPAM state exist: %v", err)
+		return
+	}
+
+	if !cniStateExists && ipamStateExists {
```

Comment:
could drop this and return immediately if `cniStateExists` above?

Reply:
similarly could you return immediately if `!ipamStateExists`

Reply:
agree on 1st case..2nd case would not happen